Examples/OISTVisualizationTool/main.swift

```diff
@@ -510,6 +506,153 @@ struct TrainRAE: ParsableCommand {
   }
 }
 
+/// PPCA with probabilistic model
+///
+/// Tracking with a Naive Bayes with RAE
+struct NaivePca: ParsableCommand {
```

Comment:
Hmm :-) I feel there is a lot of copy/paste here again, so I guess my previous request to split up in multiple functions might not have made it? By copy/pasting that mess we have created more work for us in the future.

Reply:
really don't have time, it's due to the inability to make type erased container for probability distributions

Reply:
* in short time